docs/configurations.md

```diff
@@ -0,0 +1,235 @@
+
+Select a database to see the configurations available for each storage.
+
+<div id="tabset-1">
+<div class="tab">
+  <button class="tablinks" onclick="openTab(event, 'Cassandra', 'tabset-1')" id="defaultOpen-1">Cassandra</button>
+  <button class="tablinks" onclick="openTab(event, 'CosmosDB_for_NoSQL', 'tabset-1')">CosmosDB for NoSQL</button>
+  <button class="tablinks" onclick="openTab(event, 'DynamoDB', 'tabset-1')">DynamoDB</button>
+  <button class="tablinks" onclick="openTab(event, 'JDBC_databases', 'tabset-1')">JDBC databases</button>
+</div>
+
+<div id="Cassandra" class="tabcontent" markdown="1">
+
+The following configurations are available for Cassandra:
+
+| Name                                    | Description                                                           | Default    |
+|-----------------------------------------|-----------------------------------------------------------------------|------------|
+| `scalar.db.storage`                     | `cassandra` must be specified.                                        | -          |
+| `scalar.db.contact_points`              | Comma-separated contact points.                                       |            |
+| `scalar.db.contact_port`                | Port number for all the contact points.                               |            |
+| `scalar.db.username`                    | Username to access the database.                                      |            |
+| `scalar.db.password`                    | Password to access the database.                                      |            |
+| `scalar.db.cassandra.metadata.keyspace` | Keyspace name for the namespace and table metadata used for ScalarDB. | `scalardb` |
+
+</div>
+<div id="CosmosDB_for_NoSQL" class="tabcontent" markdown="1">
+
+The following configurations are available for CosmosDB for NoSQL:
+
+| Name                                 | Description                                                                                              | Default    |
+|--------------------------------------|----------------------------------------------------------------------------------------------------------|------------|
+| `scalar.db.storage`                  | `cosmos` must be specified.                                                                              | -          |
+| `scalar.db.contact_points`           | Azure Cosmos DB for NoSQL endpoint with which ScalarDB should communicate.                               |            |
+| `scalar.db.password`                 | Either a master or read-only key used to perform authentication for accessing Azure Cosmos DB for NoSQL. |            |
+| `scalar.db.cosmos.metadata.database` | Database name for the namespace and table metadata used for ScalarDB.                                    | `scalardb` |
```

Comment:
Since this PR is for `3.8` and earlier, I think this configuration should be as follows, right?

```suggestion
| `scalar.db.cosmos.table_metadata.database` | Database name for the namespace and table metadata used for ScalarDB.                                    | `scalardb` |
```